### ISSUE TITLE
Capacitor in parallel doesn't seem to work

### ISSUE THREAD

**Issue by Ashwin-droid** [Proto-App/Proto-Android#32]
Please fix asap![Screenshot_20210406-170520_P R O T O.jpg](https://user-images.githubusercontent.com/58349846/113705154-b8b82b80-96fa-11eb-9b79-7516891317ff.jpg)

**Comment by Proto-App** [comment]
Circuit is fine, you can not use ideal capacitors without resistance in simulation because it causes shorted loop with non zero voltage. You can add very low resistance in series.

**Comment by Ashwin-droid** [comment]
Something still dosent work![Screenshot_20210407-070851_P R O T O.jpg](https://user-images.githubusercontent.com/58349846/113798136-39fdd580-9770-11eb-883d-9f8e12910947.jpg)

**Comment by Proto-App** [comment]
Strange, our internal test shows that it works just fine:
![device-2021-04-07-175242](https://user-images.githubusercontent.com/63373346/113896470-3f880980-97ca-11eb-850c-313ad31ea455.png)

**Comment by Ashwin-droid** [comment]
Maybe it is due to capistance values

**Comment by Ashwin-droid** [comment]
@Proto-App please try with my values

**Comment by Proto-App** [comment]
@Ashwin-droid 
Yes, with big capacitor values it is not correct. We will check it.
